client/verta/verta/client.py

```diff
@@ -2116,7 +2116,15 @@ def _upload_artifact(self, key, artifact_stream, part_size=64*(10**6)):
                 part_stream = six.BytesIO(file_part)
 
                 # upload part
-                response = _utils.make_request("PUT", url, self._conn, data=part_stream)
+                #     Retry connection errors, to make large multipart uploads more robust.
+                for _ in range(3):
+                    try:
+                        response = _utils.make_request("PUT", url, self._conn, data=part_stream)
+                    except requests.ConnectionError:
+                        time.sleep(10)
```

Comment:
This seems way too long. Is there any specific reason for this choice?

Reply:
No particular reason at all. Should I remove the sleep?

Wanted a bit of backoff delay for connection problems, and 10s seemed small to me in comparison to how long a 64 MB part upload can take.

Reply:
Maybe 1s? Consider that the user would have to sit 30s before knowing something is wrong.